cli/src/cli_builder.rs

```diff
@@ -476,6 +476,13 @@ fn xxx<'a>(command: CliCommand) -> Option<&'a str> {
         CliCommand::NetworkingSwitchPortSettingsView => {
             Some("system networking switch-port-settings view")
         }
+        CliCommand::NetworkingSwitchPortLldpConfigView => {
+            Some("system networking lldp config view")
+        }
+        CliCommand::NetworkingSwitchPortLldpConfigUpdate => {
+            Some("system networking lldp config update")
+        }
```

Comment:
```suggestion
        CliCommand::NetworkingSwitchPortLldpConfigView => {
            Some("system networking lldp view")
        }
        CliCommand::NetworkingSwitchPortLldpConfigUpdate => {
            Some("system networking lldp update")
        }
```

Do we think that `config` is an important subcommand or can we just view `lldp` as the thing that we're viewing and updating?

Reply:
I think "config" would differ from other, similar subcommands. Let's remove it. I expect we'll be reorganizing with @internet-diglett's work...